functionary/ui/scss/custom.scss

```diff
@@ -0,0 +1,132 @@
+// 1. Include functions first (so you can manipulate colors, SVGs, calc, etc)
+@import "./functions";
+@import "./variables";
+@import "./mixins";
+@import "./maps";
+
+// 2. Include any default variable overrides here
+$white: #FFFFFF;
+$black: #000000;
+$gunmetal: #343E40;
+
+$primary: #A05F64;
+$secondary: #C6C6C3;
+$panel: #D4D4D2;
+$body-bg: #F7F8F8;
+$tertiary: #06647F;
```

Comment:
Thoughts on switching the Functionary header panel to the tertiary color? If that breaks accessibility, then I guess it's fine. I think the tertiary's blue is a bit more welcoming than the red

Reply:
I also prefer the tertiary color as the primary color, which was the original reason I had included this section of the scss